File: backend/src/main/java/com/voiz/dto/MenuSalesDto.java
package com.voiz.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MenuSalesDto {
    private String menuName;
    private Double salesAmount;     // 실제 매출액
    private Double salesPercentage; // 비율 (필요시 사용)
    
    // 매출액만 사용하는 생성자
    public MenuSalesDto(String menuName, Double salesAmount) {
        this.menuName = menuName;
        this.salesAmount = salesAmount;
        this.salesPercentage = null;
    }
}

File: frontend/app/src/main/java/com/voyz/datas/model/dto/MenuSalesDto.kt
package com.voyz.datas.model.dto

import com.google.gson.annotations.SerializedName

data class MenuSalesDto(
    @SerializedName("menuName") val name: String,         // 메뉴 이름
    @SerializedName("salesAmount") val salesAmount: Double,  // 실제 매출액
    @SerializedName("salesPercentage") val percentage: Double? = null  // 비율 (선택적)
) {
    // 기존 코드와의 호환성을 위한 계산 속성
    val count: Float get() = salesAmount.toFloat()
}